### PR TITLE
Support `serverStatus`.

### LINKED ISSUE #156: Support `serverStatus`

### PR DESCRIPTION
**Pull request by ekalinin**
Closes #156 

Output example:

```sh
test> db.serverStatus()
{
  host: 'demo-ubuntu',
  version: '5.0.42',
  process: 'ferretdb-testcover',
  pid: Long("1900"),
  uptime: Long("2"),
  uptimeMillis: Long("2426"),
  uptimeEstimate: Long("2"),
  localTime: ISODate("2022-01-26T19:24:32.956Z"),
  catalogStats: {
    collections: 0,
    capped: 0,
    timeseries: 0,
    views: 0,
    internalCollections: 0,
    internalViews: 0
  },
  freeMonitoring: { state: 'disabled' },
  ok: 1
}

````

### CODE REVIEW

File: internal/clientconn/conn.go [hunk 1]
```diff
@@ -69,6 +69,7 @@ type newConnOpts struct {
 	proxyAddr       string
 	mode            Mode
 	handlersMetrics *handlers.Metrics
+	startUnixTime   int64
```
Comment (AlekSi) [inline internal/clientconn/conn.go:72]: I think time manipulation will be easier if we store original `time.Time` value

File: internal/handlers/msg_serverstatus.go [hunk 2]
```diff
@@ -16,20 +16,70 @@ package handlers
...
+	if err != nil {
+		return nil, lazyerrors.Error(err)
+	}
+	exec, err := os.Executable()
+	if err != nil {
+		return nil, lazyerrors.Error(err)
+	}
+
+	uptimeMillis := time.Now().UnixMilli() - h.startUnixTime
+	uptimeSecs := uptimeMillis / 1_000
```
Comment (AlekSi) [inline internal/handlers/msg_serverstatus.go:52]: With comment above, that will be just `time.Since(h.startTime).Seconds()`

**Review by AlekSi — DISMISSED** [review]

File: internal/clientconn/conn.go (hunk 1 above)
Comment (ekalinin) [reply internal/clientconn/conn.go]: Good point. Fixed.

File: internal/handlers/msg_serverstatus.go (hunk 2 above)
Comment (ekalinin) [reply internal/handlers/msg_serverstatus.go]: Agreed. Thanks!

**Review by AlekSi — APPROVED** [review]

**Comment by AlekSi** [comment]
Thank you!

**Pull request merged**